src/BattleServer/berries.cpp

```diff
@@ -272,6 +272,10 @@ struct BMPinchStat : public BMPinch
         tp(p, s, b);
     }
 
+    static void asc(int p, int s, BS &b) {
```

Comment:
I assume that the compiler will complain because `int s` is unused. If it is the case, just change `int s` to `int`